webpages/error/unsupported-browser.js

```diff
@@ -0,0 +1,52 @@
+const iframe = document.querySelector("iframe");
+
+try {
+  const uiLanguage = chrome.i18n.getUILanguage();
+  const localeSlash = uiLanguage.startsWith("en") ? "" : `${uiLanguage.split("-")[0]}/`;
+  iframe.src = `https://scratchaddons.com/${localeSlash}unsupported-browser/`;
+} catch (err) {
+  iframe.src = "https://scratchaddons.com/unsupported-browser/";
+}
+
+const IFRAME_ORIGIN = "https://scratchaddons.com";
+
+window.onmessage = async (e) => {
+  if (e.origin !== IFRAME_ORIGIN) return;
+
+  const message = e.data;
+  console.log("[SA] Message received by extension", message);
+
+  if (message.msgType) {
+    if (message.msgType === "getVersionInfo") {
+      // Reply back to the website with the extension version
+      iframe.contentWindow.postMessage({ versionInfo: chrome.runtime.getManifest().version }, IFRAME_ORIGIN);
+    }
+    if (message.msgType === "pageTitleInfo") {
+      // Website wants us to set the tab title
+      document.title = message.msgContent;
+    }
+
+    if (["exportSettingsAsFile", "openSettingsAsTab"].includes(message.msgType)) {
+      // See webpages/settings/index.js for main import/export settings code
+      const { serializeSettings } = await import("../settings/settings-utils.js");
+      const downloadBlob = (await import("../../libraries/common/cs/download-blob.js")).default;
+
+      // Export extension settings as a .json file
+      if (message.msgType === "exportSettingsAsFile") {
```

Comment:
On Chromium 79 I got an unexpected token error on this line. Firefox 85 and Chromium 104 both work as expected.

Reply:
Do you have more info on the error? Possibly async/await not properly supported?

Reply:
It's actually on line 35 of `minify-settings.js`, not this file. I haven't checked what's causing it yet.

Reply:
https://github.com/WorldLanguages/ScratchAddons/blob/029665784935ba0681d9e2efa0658f6ada2644ab/libraries/common/minify-settings.js#L35

Reply:
Chrome 80 is also unaffected so this isn't a problem.